<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criminisi Inpainting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Settings</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Patch Size</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtPatchSize">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>40</y>
             <width>41</width>
             <height>27</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Sets the width/height of a square patch</string>
           </property>
           <property name="text">
            <string>20</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnInpaint">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>100</y>
             <width>97</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Inpaint</string>
           </property>
          </widget>
          <widget class="QProgressBar" name="progressBar">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>140</y>
             <width>118</width>
             <height>23</height>
            </rect>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkDebugMessages">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>70</y>
             <width>131</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Debug Messagse</string>
           </property>
          </widget>
          <widget class="QSlider" name="sldAlpha">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>210</y>
             <width>160</width>
             <height>19</height>
            </rect>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>210</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Alpha</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblAlpha">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>230</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnStop">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>97</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnReset">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>170</y>
             <width>97</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkDebugImages">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Debug Images</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>300</y>
             <width>191</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>Priority = Conf + alpha * ColorData + beta * DepthData</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>250</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Beta</string>
           </property>
          </widget>
          <widget class="QSlider" name="sldBeta">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>250</y>
             <width>160</width>
             <height>19</height>
            </rect>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="lblBeta">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>270</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Difference</string>
          </property>
          <widget class="QRadioButton" name="radDifferenceAll">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>114</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>All</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radDifferenceDepth">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Depth weighted</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="radAll255">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>114</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>All255</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Display</string>
          </property>
          <widget class="QCheckBox" name="chkPriority">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>50</y>
             <width>95</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Priority</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkConfidence">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>80</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Confidence</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkImage">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>20</y>
             <width>95</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Image</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkBoundary">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>110</y>
             <width>95</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Boundary</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkIsophotes">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>140</y>
             <width>95</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Isophotes</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkData">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>170</y>
             <width>95</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkBoundaryNormals">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>200</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Boundary normals</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkMask">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>230</y>
             <width>95</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Mask</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKWidget" name="qvtkWidget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLeft_Pane">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenMask"/>
    <addaction name="actionOpenMaskInverted"/>
    <addaction name="actionFlipImage"/>
    <addaction name="actionSaveResult"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuLeft_Pane"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSavePointsLeft">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionSavePointsRight">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionOpenImageRight">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionLoadPointsLeft">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionLoadPointsRight">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenPointCloudLeft">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionOpenPointCloudRight">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionFlipImage">
   <property name="text">
    <string>Flip Image</string>
   </property>
  </action>
  <action name="actionFlipRightImage">
   <property name="text">
    <string>Flip Right Image</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionOpenMask">
   <property name="text">
    <string>Open Mask</string>
   </property>
  </action>
  <action name="actionOpenMaskInverted">
   <property name="text">
    <string>Open Mask Inverted</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sldAlpha</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lblAlpha</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldBeta</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lblBeta</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
